UnityProject/Assets/Scripts/Messages/Server/VariableViewer/UpdateClientValue.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System.Reflection;
+using UnityEngine;
+
+public class UpdateClientValue : ServerMessage
+{
+	public string Newvalue;
+	public string ValueName;
+	public string MonoBehaviourName;
+	public uint GameObject;
```

Comment:
This naming is a bit confusing to read
```suggestion
	public uint GameObjectNetId;
```